sklearn/tests/test_isotonic.py

```diff
@@ -429,7 +430,7 @@ def test_fast_predict():
     X_train = 20.0 * rng.rand(n_samples) - 10
     y_train = np.less(
         rng.rand(n_samples),
-        1.0 / (1.0 + np.exp(-X_train))
+        expit(X_train)
     ).astype('int64')
```

Comment:
Again, if it fits in less than 80 columns, please collapse into a single line expression.